libs/ts/config-types/src/node-config/types.ts

```diff
@@ -39,15 +39,22 @@ const PublishingCriteriaSchema = S.Struct({
   peg_tolerance_percentage: S.Number,
 });
 
+const ContractSchema = S.Struct({
+  name: S.String,
+  address: S.NullishOr(S.String),
```

Comment:
In what case we have contract name but no address?

Reply:
Contract exists, but it's not deployed. When the sequencers deploys the contract, it sets the address

Reply:
I think it never deploys. We do all deployments with the hardhat script, no?